Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -158,7 +158,6 @@ public final class ViewAnnotationManager {
         if let featureId = creationOptions.associatedFeatureId {
             viewsByFeatureIds[featureId] = view
         }
-        containerView.addSubview(view)
```

Comment:
Would the removal of this line generate the "Superview changed for annotation view. Use `ViewAnnotationManager.remove(_ view: UIView)` instead to remove it from the layout calculation." warning?

Reply:
Yes, this warning is generated when this line is removed.